crates/kornia-linalg/Cargo.toml

```diff
@@ -19,6 +19,7 @@ libm = ["glam/libm"]
 
 [dependencies]
 glam = { version = "0.30.0", default-features = false }
+nalgebra = "0.32"
```

Comment:
i'd keep kornia-linalg simple with glam only

Reply:
Yea, I'd like to have the same but our SVD isn't accurate enough for `umeyamma`. Best to address this in a separate PR.